pom.xml

```diff
@@ -186,7 +176,6 @@
         <ojdbc.version>12.1.0.2</ojdbc.version>
         <dbunit.version>2.5.4</dbunit.version>
         <spring.test.dbunit.version>1.3.0</spring.test.dbunit.version>
-        <bouncycastle.version>1.59</bouncycastle.version>
```

Comment:
- [ ] Is not domain/pom.xml lacked?

Reply:
I forgot the correspondence.
d8ef35a90129040d69e1b5a9e6b0e26c64d4cdff